File: app/src/main/java/com/evaluation/userlist/MainActivity.java
package com.evaluation.userlist;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import com.evaluation.dagger.data.DataComponent;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        DataComponent.Injector.init();

        setContentView(R.layout.activity_main);
    }
}

File: app/src/main/java/com/evaluation/userlist/MainActivity.kt
package com.evaluation.userlist

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.evaluation.dagger.data.DataComponent
import com.evaluation.dagger.data.DataComponent.Injector.init

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        init()
        setContentView(R.layout.activity_main)
    }
}